portlets/src/main/frontend/apps/functionalConfiguration/functionalConfiguration.js

```diff
@@ -1,4 +1,13 @@
 import FunctionalConfigurationApp from './components/functionalConfiguration.vue'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { fas } from '@fortawesome/free-solid-svg-icons'
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+Vue.config.productionTip = false
+library.add(fas);
+Vue.component('font-awesome-icon', FontAwesomeIcon)
 
 const lang = eXo.env.portal.language;
 const url = `/functional-configuration-portlets/vuesLocales/locale_${lang}.json`;
```

Comment:
If lang is different from fr or en, the portlet does not display.
Can we add a test to have a fallback to en in this case ?